tools/build_defs/pkg/archive.py

```diff
@@ -115,7 +115,8 @@ def __init__(self,
                name,
                compression='',
                root_directory='./',
-               default_mtime=None):
+               default_mtime=None,
+               preserve_tar_mtimes=False):
```

Comment:
As discussed, let's make the default True.

Reply:
Nope. Should be false so we don't get into the infinite incompatible flags dance.